.github/workflows/main.yml

```diff
@@ -2,8 +2,62 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  test:
-    name: Test
+  buildlibc:
+    name: Build libc
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - name: Install clang (Windows)
+      shell: cmd
+      run: |
+        powershell -Command "$ProgressPreference = 'SilentlyContinue'; iwr -outf %TEMP%\LLVM-8.0.0-win64.exe https://rust-lang-ci2.s3.amazonaws.com/rust-ci-mirror/LLVM-8.0.0-win64.exe"
```

Comment:
Can you remember why we can't use releases.llvm.org for windows?  I seem to remember there was a bug?  I wonder if updating to 9.0.0 would fix the issue?

Reply:
idk, im just trying to translate the stuff thats on azure right now. Lets try upgrading it to 9.0.0 after all this stuff works.

currently stuck on an error about llvm-ar not being found (i think?) on windows, but I have to run for the night before fixing that

Reply:
Ended up switching to the llvm release after all, but kept everything at 8.0.0 for now.